package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Legal Aid",
   "main": "index.js",
   "scripts": {
-    "lint": "gulp eslint",
+    "lint": "node node_modules/gulp/bin/gulp eslint",
```

Comment:
I was trying was to make sure the `npm lint` command works even if you don't have gulp installed globally